v2/contribute/decisions/session/0030-get-session-from-string.mdx

```diff
@@ -0,0 +1,42 @@
+---
+id: "0030"
+title: Add a new verifyAccessToken function that takes the token as a string
+hide_title: true
+---
+
+import DecisionHeader from "/src/components/decisionLogs/DecisionHeader"
+import ArgumentPro from "/src/components/decisionLogs/ArgumentPro"
+import ArgumentCon from "/src/components/decisionLogs/ArgumentCon"
+import ArgumentNeut from "/src/components/decisionLogs/ArgumentNeut"
+
+# Add a new verifyAccessToken function that takes the token as a string
+
+<DecisionHeader status="proposed" lastUpdate="2022-12-13" created="2022-12-13" deciders={["rishabhpoddar", "porcellus"]} proposedBy={["porcellus"]} />
+
+## Context and Problem Statement
+
+In some cases, the BE may need to validate access tokens based on the string only, with no access to the request/response objects.
+
+## Considered Options
+
+* Recommend using a jwt verification lib
+* Add a new `verifyAccessToken` function that takes the token as a string
+
+## Decision Outcome
+
+We've decided to add a new `verifyAccessToken` function that takes the token as a string:
+- This will be very similar to the existing `getSession`/`verifySession`
+- It will need to add a new type that makes it explicit that you can't update the session on the FE through it.
+
+### Recommend using a jwt verification lib
+
```

Comment:
also create ADRs for new createNewSessionManually and refreshSessionManually which also don't depend on req / res

Reply:
Or we could call it createNewSessionWithoutModifyingResponse and refreshSessionWithoutModifyingResponse